### PR TITLE
CI for both arms

### PR DESCRIPTION
**Pull request by achamayou**
Picked armhf, for no particular reason. Is that ok, or do we want both armel and armhf?

### CODE REVIEW

**Comment by achamayou** [comment]
So for some reason, clang-9 on armhf is not happy with -march=native:

```

/usr/bin/clang++-9  -DIS_ADDRESS_SPACE_CONSTRAINED -DSNMALLOC_CI_BUILD -DSNMALLOC_QEMU_WORKAROUND -DUSE_POSIX_COMMIT_CHECKS -DUSE_SNMALLOC_STATS -I../src -std=c++17 -stdlib=libstdc++ -g   -Wsign-conversion -Wall -Wextra -Werror -Wundef -fno-exceptions -fno-rtti -g -ftls-model=initial-exec -fomit-frame-pointer -march=native -std=gnu++1z -MD -MT CMakeFiles/func-two_alloc_types-1.dir/src/test/func/two_alloc_types/alloc2.cc.o -MF CMakeFiles/func-two_alloc_types-1.dir/src/test/func/two_alloc_types/alloc2.cc.o.d -o CMakeFiles/func-two_alloc_types-1.dir/src/test/func/two_alloc_types/alloc2.cc.o -c ../src/test/func/two_alloc_types/alloc2.cc
clang: error: the clang compiler does not support '-march=native
```

File: CMakeLists.txt [hunk 1]
```diff
@@ -145,12 +147,14 @@ if(NOT DEFINED SNMALLOC_ONLY_HEADER_LIBRARY)
...
   else()
     add_compile_options(-fno-exceptions -fno-rtti -g -ftls-model=initial-exec -fomit-frame-pointer)
-    if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
-     add_compile_options(-march=native)
-    elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86")
-     add_compile_options(-march=native)
-    elseif(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
-     add_compile_options(-march=native)
+
+    if((${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64") OR
```
Comment (achamayou) [inline CMakeLists.txt]: @mjp I believe we can get rid of this check and just rely on whether the compiler supports -march=native, what do you think?

Comment (mjp41) [reply CMakeLists.txt]: Nice if it works ;-)

Comment (achamayou) [reply CMakeLists.txt]: It does... so far? :)

**Pull request merged**